src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,142 @@
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/transition';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// These Tooltip styles will be inlined in a Gutenberg Custom HTML block
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+///
+/// 1. Horizontally centers tooltip
+/// 2. Positions tooltip vertically (below button)
+///
+
+._c-tooltip {
+  background: $tooltip-color-background;
+  border: ms.step(-4) solid $tooltip-color-background;
+  border-radius: size.$border-radius-medium;
+  box-shadow: 0 0 0 size.$edge-medium $tooltip-color-border,
+    size.$edge-large size.$edge-large 0 size.$edge-medium #{sass-color.change(
+        color.$brand-primary-darker,
+        $alpha: 0.2
+      )};
+  color: var(--theme-color-text-emphasis);
+  inset-block-start: 100%; // 2
+  inset-inline-start: 50%; // 1
+  opacity: 0;
+  position: absolute; // 1
+  transform: translate(-50%, 0); // 1, 2
+  white-space: nowrap;
+  z-index: z-index.$alert;
+
+  ///
+  /// Tooltip arrows (background and foreground)
+  ///
+  /// 1. Horizontally center arrows
+  ///
+
+  &::before,
+  &::after {
+    border-style: solid;
+    content: '';
+    inset-inline-start: 50%; // 1
+    position: absolute; // 1
+    transform: translateX(-50%); // 1
+  }
+
+  ///
+  /// Background tooltip arrow (darker color)
+  ///
+  /// 1. Combined with foreground arrow, matches border-width of tooltip body
+  /// 2. Position arrow above tooltip body
+  ///
+
+  &::before {
+    border-color: transparent transparent $tooltip-color-border;
+    border-width: 13px; // 1
+    inset-block-start: -35px; // 2
+  }
+
+  ///
+  /// Background tooltip arrow (lighter color)
+  ///
+  /// 1. Combined with background arrow, matches border-width of tooltip body
+  /// 2. Position arrow above tooltip body
+  ///
+
+  &::after {
+    border-color: transparent transparent $tooltip-color-background;
+    border-width: 10px; // 1
+    inset-block-start: -28px; // 2
```

Comment:
🤔 should we be using `rem`/`em` units here? Does this work well wth user scaling/zooming when we use px?

Reply:
This is a total nit, and I'm not sure if this is actually more readable or not so feel free to ignore. But I wonder if this could be made more DRY/readable using custom props? Something like this?

```scss
// Add two arrows to the top of tooltips
&:before,
&:after {
    border-color: transparent transparent var(--arrow-color, $tooltip-color-border);
    border-width: var(--arrow-size, 13px);
    inset-block-start: var(--arrow-offset, -35px);
}

// Customize the color, size, and placement of the second arrow
&:after {
    --arrow-color: $tooltip-color-background;
    --arrow-size: 10px;
    --arrow-offset: -28px;
}
```

(Some of the Sass variables may need to be wrapped like so to work. I can't remember for sure: `#{$sass-var}`)

Feel free to ignore this if it feels less readable. I'm not sure...

Reply:
> 🤔 should we be using rem/em units here? Does this work well wth user scaling/zooming when we use px?

OMG, of course, using `rem`/`em` makes more sense, so it can handle zooming. Thank you! 🙏🏽